package.json

```diff
@@ -66,26 +66,26 @@
         "@hapi/boom": "10.0.1",
         "@hapi/catbox-memory": "5.0.1",
         "@hapi/good": "9.0.1",
-        "@hapi/hapi": "21.3.2",
+        "@hapi/hapi": "21.3.10",
         "@hapi/inert": "7.1.0",
         "@hapi/vision": "7.0.3",
         "@mojaloop/central-ledger": "v17.7.2",
         "@mojaloop/central-services-database": "10.7.0",
         "@mojaloop/central-services-error-handling": "12.0.7",
```

Comment:
I think need to add dependency of @mojaloop/sdk-standard-components required by "@mojaloop/central-services-error-handling

Reply:
@vijayg10 any comments here?